server/helpers/isBoolStr.js

```diff
@@ -0,0 +1,5 @@
+const boolStrings = ['true', 'false']
```

Comment:
I would not put the helper in a separate file since it is used only in one place and has quite a specific usecase